streamer/src/nonblocking/recvmmsg.rs

```diff
@@ -57,16 +57,30 @@ pub async fn recv_mmsg_exact(
 mod tests {
     use {
         crate::{nonblocking::recvmmsg::*, packet::PACKET_DATA_SIZE},
+        solana_net_utils::{bind_to_async, bind_to_localhost_async},
         std::{net::SocketAddr, time::Instant},
         tokio::net::UdpSocket,
     };
 
     type TestConfig = (UdpSocket, SocketAddr, UdpSocket, SocketAddr);
 
     async fn test_setup_reader_sender(ip_str: &str) -> io::Result<TestConfig> {
-        let reader = UdpSocket::bind(ip_str).await?;
+        let sock_addr: SocketAddr = ip_str
+            .parse()
+            .map_err(|e| io::Error::new(io::ErrorKind::InvalidInput, e))?;
+        let reader = bind_to_async(
+            sock_addr.ip(),
+            /*port*/ sock_addr.port(),
+            /*reuseport:*/ false,
+        )
+        .await?;
         let addr = reader.local_addr()?;
-        let sender = UdpSocket::bind(ip_str).await?;
+        let sender = bind_to_async(
+            sock_addr.ip(),
+            /*port*/ sock_addr.port(),
```

Comment:
Same as above - I think `sock_addr.port()` is telling enough that we don't need the `/*port*/` comment